client/state/themes/selectors/should-hide-try-and-customize.js

```diff
@@ -0,0 +1,34 @@
+import { isUserLoggedIn } from 'calypso/state/current-user/selectors';
+import { canCurrentUser } from 'calypso/state/selectors/can-current-user';
+import { isJetpackSite, isJetpackSiteMultiSite } from 'calypso/state/sites/selectors';
+import {
+	isPremiumThemeAvailable,
+	isThemeActive,
+	isThemeGutenbergFirst,
+	isThemePremium,
+} from 'calypso/state/themes/selectors';
+
+import 'calypso/state/themes/init';
+
+/**
+ * Returns whether we should hide the "Try & Customize" action for a theme.
+ *
+ * @param {object} state   Global state tree
+ * @param {string} themeId Theme ID to activate in the site.
+ * @param {string} siteId  Site ID.
+ * @returns {boolean}      True if the theme should hide the Try & Customize action. Otherwise, false.
+ */
+export function shouldHideTryAndCustomize( state, themeId, siteId ) {
```

Comment:
Can this be named in the positive `shouldShow...`, for consistency and less mental work?

Reply:
I think I've done this successfully; the original logic was pretty rough to grok so this should be thoroughly tested with all site types to make sure I haven't missed an edge case.

Reply:
It's much clearer now too. <3